### PR TITLE
Armoured Amish Suit Weakened

### PR DESCRIPTION
**Pull request by ghost**
Made it only as strong as security jumpsuits and syndicate turtlenecks.

[consistency]

:cl:
 * tweak: armoured amish suit no longer the buffest under suit

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
oy vey stop that

**Comment by BomberBro** [comment]
imo it should still be stronger than the two, but only slightly, since it requires more effort than hacking the sec checkpoint door or the suitvend (unless I'm retarded and you can't get turtlenecks from those).

**Comment by ghost** [comment]
I've had people propose it should have some unique features, even if it isn't as overblown as it was before. Maybe have it as some experimental armour for a bartender/assassin. The main issue is balance as if you stack the armoured amish with a HoS coat or reinforced mining rigsuit you are invulnerable to simple mobs.

Maybe some extra bullet resistance would be ok.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -37,9 +37,9 @@
 	species_fit = list(GREY_SHAPED)
 
 /obj/item/clothing/under/sl_suit/armored
-	desc = "It's a very amish looking suit. This one looks armored."
+	desc = "It's a very amish looking suit. This one has a tag. It says "Gorlex" in cursive."
```
Comment (MadmanMartian) [inline code/modules/clothing/under/miscellaneous.dm]: It says "Gorlex" in cursive."

Change it to It says It says \\"Gorlex\\" in cursive."

Comment (ghost) [reply code/modules/clothing/under/miscellaneous.dm]: Alright, will do. Any idea why the check is failing, exactly?

Comment (ghost) [reply code/modules/clothing/under/miscellaneous.dm]: I guess that was it. Thanks. Can't put use -"- in code frivolously, should have realised.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -37,9 +37,9 @@
 	species_fit = list(GREY_SHAPED)
 
 /obj/item/clothing/under/sl_suit/armored
-	desc = "It's a very amish looking suit. This one looks armored."
+	desc = "It's a very amish looking suit. This one has a tag. It says  \"Gorlex\" in cursive."
```
Comment (jknpj) [inline code/modules/clothing/under/miscellaneous.dm]: too many spaces

**Pull request merged**